modules/cluster-logging-exported-fields-kubernetes.adoc

```diff
@@ -1,15 +1,18 @@
 [id="cluster-logging-exported-fields-kubernetes_{context}"]
-== Kubernetes
 
-// Normally, the preceding title would be an H1 prefixed with an `=`. However, because the following content is auto-generated at https://github.com/ViaQ/documentation/blob/main/src/data_model/public/kubernetes.part.adoc and pasted here, it is more efficient to use it as-is with no modifications. Therefore, to "realign" the content, I am going to prefix the title with `==` and use `include::modules/cluster-logging-exported-fields-kubernetes.adoc[leveloffset=0]` in the assembly file.
```

Comment:
Let me know offline if you want me to modify the generator to make this more convenient.